openedx/features/course_experience/utils.py

```diff
@@ -133,6 +136,10 @@ def mark_last_accessed(user, course_key, block):
             'video',
             'discussion'
         ]
+        custom_block_types =  configuration_helpers.get_value(CUSTOM_BLOCK_TYPES)
```

Comment:
Why are we adding additional custom types?

Reply:
Completion django app uses this block types to track which activities are completed and we need to track more kind of activities, for example ora-xblock, so we have to add these from a config setting,in order to the user could define which kind of activities are wanted